Source/astcenc_weight_align.cpp

```diff
@@ -63,11 +73,12 @@ static const float angular_steppings[44] = {
 	20.0f, 21.0f, 22.0f, 23.0f,
 	24.0f, 25.0f, 26.0f, 27.0f,
 	28.0f, 29.0f, 30.0f, 31.0f,
-	32.0f, 33.0f, 34.0f, 35.0f
+	32.0f, 33.0f, 34.0f, 35.0f,
+#if ANGULAR_STEPS >= 48
+    32.0f, 33.0f, 34.0f, 35.0f,
```

Comment:
Duplicating this row ensures we have the same results on 4 and 8 wide, which is important for users who want stable output across instruction sets. Seems a shame from a performance point of view, as it's dummy work.

Seems worth a comment to explain why we do it this way though.

Reply:
Added a comment in a8c2e89